src/useCKEditor.ts

```diff
@@ -173,6 +179,25 @@ function useCKEditor<EditorEvent extends string>( {
 						if ( isInline && !isReadOnly ) {
 							editor.setReadOnly( false );
 						}
+
+						/**
+						 * Sets initial content of editor's instance if provided.
+						 */
+						if ( initContentRef.current ) {
+							editor.setData( initContentRef.current, {
+								/**
+								 * Prevents undo icon flickering.
+								 */
+								noSnapshot: true,
+
+								/**
+								 * Resets undo stack.
+								 */
+								callback: () => {
+									editor.resetUndo();
+								}
```

Comment:
The callback can be probably safely omitted as it's the first `setData` and there's already `noSnapshot` option set to `true`. Due to that undo stack should be empty.

Reply:
Indeed, `callback` option is unnecessary. I've removed it and observed no issues.